core/modules/taxonomy/taxonomy.module

```diff
@@ -173,6 +184,26 @@ function taxonomy_field_extra_fields() {
         ),
       ),
     );
+
+    // Add also the 'language' select if Language module is enabled and the
```

Comment:
- "Also add..."
- Perhaps rephrase to have the conditional at the beginning of the sentence:

```
// If the Language module is enabled, and the bundle has multilingual support,
// then also add the `language` select.
```